eventhubs-capture-synapse-delta/create-solution.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+# Strict mode, fail on any error
+set -euo pipefail
+source set-variables
+source ../streaming/synapse/job/run-synapse-pipeline.sh
+
+on_error() {
+    set +e
+    echo "There was an error, execution halted" >&2
+    echo "Error at line $1"
+    exit 1
+}
+
+trap 'on_error $LINENO' ERR
+
+export SQL_ADMIN_USER="sasesssynadmin"
+export SPARK_VERSION="2.4"
+export FILE_SYSTEM=streamingatscale
+export SYNAPSE_SPARKPOOL="sasesssparkpool"
+
+export PREFIX=''
+export LOCATION="eastus"
+export TESTTYPE="1"
+export STEPS="CIPTMV"
+export WAITVERIFICATION=true
+
+usage() {
+    echo "Usage: $0 -d <deployment-name> $1 -p <sparkpool-sql-password> [-s <steps>] [-t <test-type>] [-l <location>] [-w <wait-verfication>]"
```

Comment:
please document this `-w` parameter. (Though my preference would be to remove it completely, for simplicity, the user can just cancel the script execution)